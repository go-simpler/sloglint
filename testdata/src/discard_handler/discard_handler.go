package discard_handler

import (
	"io"
	"log/slog"
)

func _() {
	_ = slog.NewTextHandler(io.Discard, nil) // want `use slog.DiscardHandler instead`
	_ = slog.NewJSONHandler(io.Discard, nil) // want `use slog.DiscardHandler instead`
}
