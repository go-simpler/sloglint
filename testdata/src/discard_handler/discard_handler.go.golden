package discard_handler

import (
	"io"
	"log/slog"
)

func _() {
	_ = slog.DiscardHandler() // want `use slog.DiscardHandler instead`
	_ = slog.DiscardHandler() // want `use slog.DiscardHandler instead`
}
